BeforeBuild.cmd

```diff
@@ -0,0 +1,35 @@
+::Configuration for AppVeyor build platform
+
+@ECHO off
+SETLOCAL
+
+if NOT exist "GeneratedKey.snk" (
+sn -k GeneratedKey.snk
```

Comment:
This runs on the AppVeyor server, right? That means that `sn.exe` has to be on the execution path... ah, I see that you set up the Visual Studio dev environment in the `.yml` file. Good.